pkg/processor/build/builder.go

```diff
@@ -623,6 +624,11 @@ func (b *Builder) createProcessorDockerfile() (string, error) {
 		return "", errors.Wrap(err, "Could not find a proper base image for processor")
 	}
 
+	b.options.FunctionConfig.Spec.Build.Commands, err = b.preprocessBuildCommands(b.options.FunctionConfig.Spec.Build.Commands, "")
```

Comment:
As I mentioned in Slack - _don't_ overwrite b.options.FunctionConfig.Spec.Build.Commands. Do this on a copy.
We do not want to mutate and lose the original input.